server/backend/testspg/crud/test_crud_user_role.py

```diff
@@ -102,15 +98,15 @@ def test_get_use_role_by_user_id_and_role_id(db_pg_connection: Session):
 
     # find it
     found_user_role_xref = crud_user_role.get_use_role_by_user_id_and_role_id(
-        db_pg_connection,
+        db_pg_session,
         TEST_USER_ID,
         TEST_FOM_DEV_REVIEWER_ROLE_ID,
     )
     assert found_user_role_xref is not None
 
     # cleanup
     crud_user_role.delete_fam_user_role_assignment(
```

Comment:
just curious, do we still need to call this delete method? or it can auto rollback?

Reply:
Looks like I have missed tow of them. Thanks for spotting that.